src/Bar.js

```diff
@@ -117,7 +117,7 @@ export default class BarChart extends Component {
     let textStyle = fontAdapt(options.axisX.label)
 
     let lines = chart.curves.map(function (c, i) {
-      let color = this.color(i % 3)
+      let color = _.isArray(this.props.options.color) ? this.props.options.color[i] : this.color(i % 3)
```

Comment:
@moorthysubu I was hoping @mlabrum would address changes requested above to get this approved and merged in. Since it has been awhile without a response from @mlabrum, are you willing to fix and address requested changes above and submit a new pull request?